.eslintrc

```diff
@@ -42,6 +42,10 @@ rules:
   react/prop-types: off # possibly reinstate
   react/sort-comp: off # possibly reinstate
   jsx-a11y/no-static-element-interactions: off
+  import/extensions: off
+  import/first: off
+  import/newline-after-import: off
+  import/no-extraneous-dependencies: off
```

Comment:
Do we have a plan for turning them back on? Or are we ok with indefinitely disabling them?

Reply:
I'm going to try fixing and removing them in this PR - will see if that gets too tedious.

Reply:
Done, wasn't too bad. Relevant commits: 54dfb22d0f4927c6b3b8adb506df5aa9266989ee...cb853ed56c90507f2e3360e37c2c7dce9c68ff77